demo/ff/src/guichanffdemo.cpp

```diff
@@ -45,6 +45,7 @@
 #include <guisan.hpp>
 #include "ffdemo.hpp"
 
+#undef main
```

Comment:
Should not be needed, I would say misusage of SDL_main.

Reply:
Indeed. Fixed with an updated commit, and I've also updated the project files to use the right SDL2main libs (debug or release), accordingly.